core/src/main/scala/io/chrisdavenport/epimetheus/Gauge.scala

```diff
@@ -79,17 +79,17 @@ sealed abstract class Gauge[F[_]]{
 object Gauge {
 
   // Convenience
-  def incIn[E, F[_]: Bracket[?[_], E], A](g: Gauge[F], fa: F[A]): F[A] = 
-    Bracket[F, E].bracket(g.inc)(_ => fa)(_ => g.dec)
+  def incIn[E, F[_]: MonadCancel[*[_], E], A](g: Gauge[F], fa: F[A]): F[A] =
```

Comment:
I think it's even a pattern in CE3 itself to use `MonadCancel[*[_], _]` in these situations (if E isn't part of the signature otherwise).

Also I think `*[_]` isn't supported by `-Ykind-projector` in dotty so might just want to pass an explicit implicit :)

Reply:
Hope I understood `explicit implicit` thing correctly here :)

Reply:
Ah, i meant passing the MonadCancel instamce using an `implicit` clause, as opposed to a context bound. Sorry for the confusion.

That way it'd be `MonadCancel[F, _]` I think.

Reply:
Yes, this is how I read this and moved `MonadCancel` to implicit parameter with the second type parameter unlocked.